Updated the Readme for tutorial-v2 (#90)
PureStake/moonbeam: README.md

## Install (linux)

### Get the code
Get the tutorial specific tag of the PureStake/Moonbeam repo:
```bash
git clone -b tutorial-v2 https://github.com/PureStake/moonbeam
cd moonbeam
```

### Setting up enviroment

Install Substrate pre-requisites (including Rust):  
```bash
curl https://getsubstrate.io -sSf | bash -s -- --fast
```

Run the initialization script, which checks the correct rust nightly version and adds the WASM to that specific version:
```bash
./scripts/init.sh
```

## Build Standalone
Build the corresponding binary file:

```bash
cd node/standalone
cargo build --release
```  

## Build Parachain
Build the corresponding binary file:
```bash
cargo build --release
```  

> The first build takes a long time, as it install and compiles all the necessary libraries.

Reviewer: ```suggestion
The first build takes a long time, as it compiles many dependencies.
```

"Install" isn't really the right word here. It is compiling that code to be part of our binary.

Author: Modified it thanks 👍
